Source/Patches/Core/BeltBagPatch.cs

```diff
@@ -0,0 +1,34 @@
+﻿using GameNetcodeStuff;
+using HarmonyLib;
+using PlayerDogModel_Plus.Source.Model;
+using UnityEngine;
+
+namespace PlayerDogModel_Plus.Source.Patches.Core
+{
+    [HarmonyPatch(typeof(BeltBagItem))]
+    internal class BeltBagPatch
+    {
+        [HarmonyPatch("LateUpdate")]
+        [HarmonyPostfix]
+        public static void LateUpdatePostfix(ref BeltBagItem __instance, PlayerControllerB ___playerHeldBy, bool ___isHeld, bool ___isPocketed)
+        {
+            if (___playerHeldBy != null && ___isHeld)
+            {
+                PlayerModelReplacer replacer = ___playerHeldBy.GetComponent<PlayerModelReplacer>();
+
+                if (replacer == null || !replacer.IsDog) return; // Nothing to do.
+
+                if (!___isPocketed)
+                {
+                    __instance.transform.position = replacer.itemAnchor.position;
+                    return;
+                }
+
+                Transform dogTorso = replacer.GetDogGameObject().transform.Find("Armature").Find("torso");
```

Comment:
This is just essentially hiding the belt bag when pocketed. It would be nice to change this to make it look like a lil collar though!